packages/components/organisms/f-content-cards/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/f-content-cards",
   "description": "Fozzie Content Cards",
-  "version": "4.0.0",
+  "version": "4.0.1",
```

Comment:
Should be a minor release – has too much change to be considered a patch.

Reply:
Agreed will make that change